File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
     
        
       if (nums.size() == 0) return 0;
  
  int walker = 0, runner = 0, size = nums.size();
  while (runner < size) {
      if (nums[walker] == nums[runner]) ++runner;
      else {
          ++walker;
          nums[walker] = nums[runner];  
          ++runner;
            }
        }
         nums.erase(nums.begin() + walker + 1, nums.end());
  
    return nums.size();
    }
};

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        
        x = 0 
        for i in range(1,len(nums)):
            
            if nums[x] != nums[i]:
                x+=1 
                nums[x] = nums[i]
        return x+1 
        